template/report.html.tmpl

```diff
@@ -14,450 +15,596 @@
-          <th> </th>
-          <th colspan="3" style="text-align: center;">App</th>
-          <th colspan="3" style="text-align: center;">DB (Postgres)</th>
-          <th colspan="3" style="text-align: center;">Relay</th>
-        </tr>
-        <tr>
-          <th> </th>
-          <th>Before</th>
-          <th>After</th>
-          <th>Difference</th>
-          <th>Before</th>
-          <th>After</th>
-          <th>Difference</th>
-          <th>Before</th>
-          <th>After</th>
-          <th>Difference</th>
-        </tr>
+        <div class="flex flex-col mt-4">
+          <div class="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+            <div class="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+              <div class="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
+                <table class="min-w-full divide-y divide-gray-200 text-xs">
+                  <thead class="bg-gray-50">
+                    <tr>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Type
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Min
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Mean
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        50th
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        90th
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        95th
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        99th
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Max
+                      </th>
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Total
+                      </th>
+                    </tr>
+                  </thead>
+                  {{ range .Latency }}
+                  <tr>
+                    <td class="px-6 py-4">{{ .Name }}</td>
+                    {{ if .Diff }}
+                      <td class="px-6 py-2">{{ round .Metrics.Min }} <div class="text-gray-400">({{ .Diff.Min }}%)</div></td>
```

Comment:
Can we make the percentage darker than `gray-400`? It's kind of hard to see (assuming this follows the same deal as https://getbootstrap.com/docs/5.0/customize/color/). Maybe 600?

Reply:
Fine with me. Can we do these small improvements later? I am not sure I will have time